ci/jenkins/Jenkinsfile_website_full

```diff
@@ -40,13 +40,7 @@ core_logic: {
 
   utils.parallel_stage('Build Docs', [
     custom_steps.docs_jekyll(),
-    custom_steps.docs_c('libmxnet'),
     custom_steps.docs_python('libmxnet'),
-    custom_steps.docs_julia('libmxnet'),
```

Comment:
After this PR was merged, none of these docs are appearing on the MXNet website anymore. Why were these removed from the build steps?

Reply:
See #18604.

Reply:
Can we revert the merge of this PR and rebuild website ASAP?

Reply:
These front ends need to be developed on mxnet 2.0 as they depend on deprecated C API. For master branch, not showing them is intended. For other versions, if there’s anything broken, then the doc build logic needs fix

Reply:
@szha - if other versions are broken, my opinion is that needs to be fixed before this PR is merged. If not, are you saying it is expected and accepted breaking change in website master branch?

Reply:
@sandeep-krishnamurthy I'm not aware of whether other versions are broken, hence the "if". The point I'm trying to make is that the other language bindings in previous versions, which are fully functional, shouldn't have their doc build logic dependent on master branch.

And yes, the master branch doc should not show language bindings until they are migrated to the new C API.

Reply:
@sandeep-krishnamurthy the problem here really is that the website defaults to the master version instead of the stable version. https://github.com/apache/incubator-mxnet/issues/18429

Reply:
Thanks. From my understanding, other version branches are not impacted, it is just the master. @ys2843 can confirm. If that is the case, what would be your recommendation for scala/java/r/c++ api in master version of website? 
I think either we should remove from website API drop down or put a page saying it is work in progress.
@aaronmarkhama what do you suggest here?

Reply:
Other versions of website except master are archived static files and their build process does not depend on any branch.